### PR TITLE
[deps] update libbluray to 1.3.2

### LINKED ISSUE #21016: Kodi crash when start playing movie inside Bluray menu
### Describe the bug
Here is a clear and concise description of what the problem is:

When playing a Bluray whin the Bluray menu the window about parent control pop up but after a few seconds later Kodi close completely. Latest Java update 321 is installed. The moving inside the Bluray menu works very good without any issue.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

### To Reproduce

1. Start movie with Bluray menu
2. Click play movie

### Debuglog

The debuglog can be found here: https://paste.kodi.tv/lesaqinadi.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name: Windows 10 Pro 21H2
 - Kodi version: v20 Alpha 1 (19.90.101)

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

### PR DESCRIPTION
**Pull request by enen92**
## Description
Libbluray 1.3.2 was released 1 month ago as an hotfix for [Oracle Java CPU for April 2022](https://code.videolan.org/videolan/libbluray/-/commit/8f26777b1ce124ff761f80ef52d6be10bcea323e).
~Windows deps is still missing in this PR.~

Also not sure if the minimal supported version should be bumped from the current 0.9.3. Since this release is similar to an hotfix I'd say we should keep the minimum requirements as is.

### CODE REVIEW

File: project/BuildDependencies/scripts/0_package.target-win10-arm.list [hunk 1]
```diff
@@ -17,7 +17,7 @@ lcms2-2.9-win10-arm-v141-20200105.7z
 libaacs-0.9.0-win10-arm-v141-20200203.7z
 libass-0.15.2-win10-arm-v142-20210920.7z
 libbdplus-0.1.2-win10-arm-v141-20200105.7z
-libbluray-1.1.2-win10-arm-v142-20200706.7z
+libbluray-1.3.2-win10-arm64-v142-20220908.7z
```
Comment (thexai) [inline project/BuildDependencies/scripts/0_package.target-win10-arm.list:20]: Seems UWP-ARM package is wrong. Should be: 
`libbluray-1.3.2-win10-arm-v142-20220908.7z`


`12:18:54 libkodi.lib(BlurayDirectory.obj) : error LNK2001: unresolved external symbol bd_init [D:\jenkins\workspace\WIN-UWP-ARM\kodi-build.arm-uwp\kodi.vcxproj]`

But seems that correct package is missing in mirrors

Comment (enen92) [reply project/BuildDependencies/scripts/0_package.target-win10-arm.list:20]: yeap I'm trying to get this to build for arm, it looks like it was disabled in the kodi-deps repo

**Review by thexai — APPROVED** [review]
Go ahead as soon as Jenkins in green

**Comment by enen92** [comment]
Jenkins build this please

**Comment by enen92** [comment]
aff, this was quite a huge PITA. Hope someday windows deps are managed in tree just like all the other platforms.
thanks to the ones who provided help along the way (fuzzard, wsnipex, paxxi, thexai).

**Pull request merged**